docs/UserGuide.md

```diff
@@ -79,71 +73,54 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME i/IC_NUMBER ag/AGE s/GENDER a/ADDRESS`
 
 <box type="tip" seamless>
 
-**Tip:** A person can have any number of tags (including 0)
+**Tip:** A person can have a note included.
 </box>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
-
-### Listing all persons : `list`
-
-Shows a list of all persons in the address book.
-
-Format: `list`
+* `add n/John Doe i/T0123456A ag/12 s/Male a/John street, block 123, #01-01`
 
-### Editing a person : `edit`
+### Adding a note : `addnote`
 
-Edits an existing person in the address book.
+Add a note to an existing person in the address book
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `addnote i/IC_NUMBER n/NOTE`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
+* Adds a note to the person with the specified `IC_NUMBER`. The IC number refers to the IC number shown in the displayed person list. The IC_NUMBER **must be the FULL IC NUMBER**.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* When adding notes, the existing note of the person will be removed i.e adding of note is not cumulative.
```

Comment:
I think the feature spec has that new notes are appended if they already exist.
![image](https://github.com/AY2324S2-CS2103T-F14-2/tp/assets/110999129/cfe26568-92ed-4f60-a22b-da4ab7aea1be)

Reply:
Fixed in #27